### DISCUSSION TITLE
Sync Changes Between stable and lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#29430] [Ideas] [answered]
I am very new to using Github so i apologize if there are any mistakes! I am also not experienced in coding so i wont be able to understand if it comes up.

If you have imported all your data from stable to lazer and for example delete a bunch of beatmaps in stable, those maps are still avalible in lazer and you would either have to delete them again in lazer or remove all beatmaps and import them from stable again. This also happens if you remove or add maps to collections, delete or add skins etc. It would be really useful if there was a option in lazer to check for changes made in stable and then automatically make lazer have the same changes so both lazer and stable have the exact same maps, collections, scores and skins.

Im assuming this is for players like me who have alot of maps, skins and scores in stable and are waiting for the switch to lazer and want stable and lazer to have the exact same contents and doesnt want to individually change lazer so that it has the same contents as stable or delete everything in lazer and import everything back which takes a very long time.

**Comment by peppy** [comment, marked as answer]
You will have to manually delete them from lazer.

There's no intention to have any kind of sync.

**Reply by ghost** [reply]
Alright, thanks for the info
